include/game/multiplayer/mp_player.h

```diff
@@ -33,6 +33,6 @@ void CreateMultiplayerPlayer(u8);
 
 void sub_8018818(void);
 
-extern u8 gUnknown_0300583C;
+extern u8 createMPAttack2Effect;
```

Comment:
Minor, but globals need to be named with `g` prefix `gCreate...`

Reply:
Also, is it a boolean?
- if yes, please make it a bool8
- if not, better remove the `Create` bit from the name, it's a bit confusing otherwise